### ISSUE TITLE
Add support for compose v2

### ISSUE THREAD

**Issue by runseb** [skippbox/kompose#19]
It is in the roadmap for libcompose.
If we manage to do it before we should submit upstream

**Comment by ngtuna** [comment]
Let me take it.

**Comment by ngtuna** [comment]
Agreed to wait for upgrading of `libcompose`. Working on kcompose's part. #https://github.com/docker/libcompose/issues/147
